openg2p_registration/models/openg2p_registration.py

```diff
@@ -680,6 +681,9 @@ def create_beneficiary_from_registration(self):
             "emergency_contact": self.emergency_contact,
             "emergency_phone": self.emergency_phone,
             "odk_batch_id": self.odk_batch_id,
+            "kyc_id": self.kyc_id,
```

Comment:
Why add this as new field? It already exists as part of registration model linked with KYCs.